conan/cli/formatters/graph/info_graph_html.py

```diff
@@ -44,7 +44,7 @@
                     <label for="show_package_type">Show package type</label>
                 </div>
                  <div>
-                    <input type="search" placeholder="Search packages..." oninput="searchPackages(this)">
+                    <input type="search" placeholder="Search packages..." oninput="searchPackages(this)" onkeydown="onSearchKeyDown(event)">
```

Comment:
what problem this debounce is solving? I haven't experienced any issue in the ``conan graph info`` html output that deserves a debounce